app/packs/entrypoints/application.js

```diff
@@ -1,19 +0,0 @@
-/* eslint no-console:0 */
```

Comment:
why remove this file? It is the standard entrypoint for Rails applications. I would keep it here as it is a convention to have it.
If you have some specific reason to remove it, let me know.